ngff-tools.md

```diff
@@ -0,0 +1,198 @@
+      <td>y</td>
+      <td>n</td>
+      <td><a href="https://github.com/InsightSoftwareConsortium/itkwidgets/issues/546">n</a></td>
+      <td>n</td>
+      <td>n?</td>
+    </tr>
+    <tr>
+      <td>multiscales factor not=2</td>
+      <td><a href="https://minio-dev.openmicroscopy.org/idr/v0.4/idr0082/9846318.zarr/0">9846318.zarr/0</a></td>
+      <td>y</td>
+      <td><a href="https://github.com/hms-dbmi/vizarr/issues/101">n</a></td>
+      <td>y</td>
+      <td>y</td>
+      <td>y</td>
+      <td>y?</td>
+    </tr>
+    <tr>
+      <td>axes (v0.3)</td>
+      <td><a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/idr0079A/9836998.zarr">9836998.zarr</a></td>
+      <td>y</td>
+      <td>y</td>
+      <td>y</td>
+      <td>y</td>
+      <td>n</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>v0.4 axes (v0.4)</td>
+      <td><a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0101A/13457227.zarr">13457227.zarr</a></td>
+      <td>y</td>
+      <td>y</td>
+      <td>y</td>
+      <td>y</td>
+      <td>y</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>3D view</td>
+      <td><a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0062A/6001240.zarr">6001240.zarr</a></td>
+      <td>y (1)</td>
+      <td>n</td>
+      <td>y</td>
+      <td>y</td>
+      <td>y</td>
+      <td>n</td>
+    </tr>
+    <tr>
+      <td>labels</td>
+      <td><a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0052A/5514375.zarr">5514375.zarr</a></td>
+      <td>y</td>
+      <td>n</td>
+      <td>y</td>
+      <td><a href="https://github.com/InsightSoftwareConsortium/itkwidgets/issues/547">n</a></td>
+      <td>y</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>HCS plate</td>
+      <td><a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0001A/2551.zarr">idr0001</a></td>
+      <td>y</td>
+      <td>y</td>
+      <td>n</td>
+      <td>n</td>
+      <td>n</td>
+      <td>y</td>
+    </tr>
+    <tr>
+      <td>bioformats2raw.layout</td>
+      <td><a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.2/idr0070A/9838562.zarr">9838562.zarr</a></td>
+      <td><a href="https://github.com/ome/napari-ome-zarr/issues/71">n</a></td>
+      <td><a href="https://github.com/hms-dbmi/vizarr/issues/149">n</a></td>
+      <td><span style="color: red">X</span></td>
+      <td>n</td>
+      <td>n</td>
+      <td>y</td>
+    </tr>
+    <tr>
+      <td>multiple 'multiscales'</td>
+      <td></td>
+      <td>n</td>
+      <td>n</td>
+      <td>n</td>
+      <td>n</td>
+      <td>n</td>
+      <td>n</td>
+    </tr>
+    <tr>
+      <td>datasets coordinateTransformations (v0.4)</td>
+      <td><a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0101A/13457537.zarr">13457537.zarr</a>
+        translate onto <a href="https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0101A/13457227.zarr">13457227.zarr</a></td>
+      <td>y (scale, translate)</td>
+      <td>n</td>
+      <td>?</td>
+      <td>y (scale)</td>
+      <td>y (scale)</td>
```

Comment:
This line, which says ``webknossos opens ?`` should say something like 

```
errors for the first image (with scale and transform)
opens for the second image (with only scale)
```
This invites a split of this line into the first image (scale & transform) and the second image (scale).

We also need a better expression for the ``n`` which at present means "does not support the feature which the particular image is trying to highlight, it opens the image but ignores the feature".